src/iconsManifest/supportedExtensions.ts

```diff
@@ -3181,6 +3181,14 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
       disabled: true,
     },
+    {
+      icon: 'preact',
+      extensions: [],
+      filenamesGlob: ['preact.config'],
```

Comment:
This is not strictly needed as there's only one possible extension `.js` but it doesn't hurt. You can see what would be the leaner option just below in the `precommit` icon.